docs/docs/_available-tags.md

```diff
@@ -0,0 +1,21 @@
+# Available tags
+
+The following is a list of available tags that can be used when contributing a sample. Click on the links to view the samples associated with the tags.
+
+Tag | Description
+-- | --
+[AI](./tags#AI) | Content dealing with AI related subjects. 
```

Comment:
Let's simplify the descriptions and remove the _Content about/dealing/involving_. They're unnecessary and make it harder to read the descriptions.